server/routes/app.ts

```diff
@@ -37,20 +38,10 @@ const readIndexFile = async (ctx: Context): Promise<Buffer> => {
     );
   }
 
-  const middleware = ctx.devMiddleware;
-  await new Promise((resolve) => middleware.waitUntilValid(resolve));
-  return new Promise((resolve, reject) => {
-    middleware.fileSystem.readFile(
-      `${ctx.webpackConfig.output.path}/index.html`,
-      (err: Error, result: Buffer) => {
-        if (err) {
-          return reject(err);
-        }
-
-        resolve(result);
-      }
-    );
-  });
+  // TODO: Should be part of the build
+  return await readFile(
+    path.join(__dirname, "../../../server/static/index.html")
+  );
```

Comment:
Actually, I forgot why I added this TODO and I even forgot what this code is doing. I think we can keep it like that and just remove the TODO. Or am I missing something?